interface/Cargo.toml

```
@@ -0,0 +1,42 @@
+[package]
+name = "solana-system-interface"
+version = "0.0.1"
+description = "Instructions and constructors for the System program"
+repository = "https://github.com/solana-program/system"
+edition = "2021"
+readme = "README.md"
+license-file = "../LICENSE"
+
+[package.metadata.docs.rs]
+targets = ["x86_64-unknown-linux-gnu", "wasm32-unknown-unknown"]
+all-features = true
+rustdoc-args = ["--cfg=docsrs"]
+
+[dependencies]
+num-traits = "0.2"
+serde = "1.0.210"
+serde_derive = "1.0.210"
+solana-decode-error = "^2.1"
+solana-frozen-abi = { version = "^2.1", optional = true }
+solana-frozen-abi-macro = { version = "^2.1", optional = true }
```

I believe this should also enable the `frozen-abi` feature on each crate
```suggestion
solana-frozen-abi = { version = "^2.1", optional = true, features = ["frozen-abi"] }
solana-frozen-abi-macro = { version = "^2.1", optional = true, features = ["frozen-abi"] }
```